Podfile.lock

```diff
@@ -51,26 +51,37 @@ PODS:
   - EmailChecker (0.1)
   - Expecta (0.3.2)
   - Fabric (1.6.7)
-  - FormatterKit (1.8.0):
-    - FormatterKit/AddressFormatter (= 1.8.0)
-    - FormatterKit/ArrayFormatter (= 1.8.0)
-    - FormatterKit/ColorFormatter (= 1.8.0)
-    - FormatterKit/LocationFormatter (= 1.8.0)
-    - FormatterKit/NameFormatter (= 1.8.0)
-    - FormatterKit/OrdinalNumberFormatter (= 1.8.0)
-    - FormatterKit/TimeIntervalFormatter (= 1.8.0)
-    - FormatterKit/UnitOfInformationFormatter (= 1.8.0)
-    - FormatterKit/URLRequestFormatter (= 1.8.0)
-  - FormatterKit/AddressFormatter (1.8.0)
-  - FormatterKit/ArrayFormatter (1.8.0)
-  - FormatterKit/ColorFormatter (1.8.0)
-  - FormatterKit/LocationFormatter (1.8.0)
-  - FormatterKit/NameFormatter (1.8.0)
-  - FormatterKit/OrdinalNumberFormatter (1.8.0)
-  - FormatterKit/TimeIntervalFormatter (1.8.0)
-  - FormatterKit/UnitOfInformationFormatter (1.8.0)
-  - FormatterKit/URLRequestFormatter (1.8.0)
-  - Gridicons (0.1)
+  - FormatterKit (1.8.1):
```

Comment:
Not sure what the deal was with formatter kit.  I'm going to revert the commit with the podfile lock since it looks like the 6.2 branch got the Gridicon fix cherry picked into it :)